content/lua/concepts/strings/terms/reverse/reverse.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: '.reverse()'
+Description: ' rearranges a provided string in reverse order'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **`.reverse()`** function takes a valid string and rearranges it in reverse order.
+
+## Syntax
+
+```lua
+string.reverse(originalString)
+
+```
+
+The **`originalString`** is the string to be reversed.
```

Comment:
```suggestion
The `originalString` is the string to be reversed.
```

Reply:
bold is only applied to the first instance of the method or concept in the entry